index.html

```diff
@@ -188,9 +188,9 @@ <h2 class="font-white thin">ROSTER</h2>
                     <div class="col d-flex p-2">
                         <!--Card-->
                         <div class="card card-cascade">
-                            <img class="img-fluid rounded-circle img-responsive roster" src="resources/profiles/zane.jpg" alt="Zane Jeffries">
+                            <img class="img-fluid rounded-circle img-responsive roster" src="resources/profiles/Chris.jpg" alt="Chris Cashin">
                             <div class="card-body text-center">
-                                <p class="font-gray">Zane Jeffries</p>
+                                <a href="https://www.instagram.com/chr1s_cash1n/" class="font-gray">Chris Cashin</p>
```

Comment:
Closing tag should be an anchor tag, not a closing paragraph tag:

`<a href="https://www.instagram.com/chr1s_cash1n/" class="font-gray">Chris Cashin</a>`